src/common.rs

```diff
@@ -2110,6 +2110,12 @@ impl Disk {
 ///     println!("{disk:?}");
 /// }
 /// ```
+///
+/// ⚠️ Note that network devices are excluded by default under Linux.
+/// To display mount points using the CIFS and NFS protocols, the `linux-netdevs`
+/// feature must be enabled. Note, however, that sysinfo may hang under certain
+/// circumstances. For example, if a CIFS or NFS share has been mounted with
+/// the _hard_ option, but the connection has an error, such as the share server has stopped.
```

Comment:
Didn't you want to add a check to prevent that? I remember suggestions done in the issue.

Reply:
You talk about the timeout thing? For that it needs a second thread, but I have not found a timeout function for Rayon. And I have no idea how this could be done then.

The question is, if this is really needed. Even tools like _df_ hangs when the mount is faulty. In a stable network it should mostly not happen, that a mount goes down.

I my option user can decide if they can life with the risk, and they have also the option to mount with _soft_.

Reply:
Ok makes sense. Let's keep it that way then. :+1: